src/templates/quote/icon.js

```diff
@@ -0,0 +1,36 @@
+import { React } from 'react';
+import PropTypes from 'prop-types';
+
+/**
+ * @param {object} props
```

Comment:
I see no props parameter? Only see destruction of propes.

Reply:
The linter yells at me if I dont do it